thoth/user_api/openapi_server.py

```diff
@@ -48,6 +50,7 @@
 
 # Expose for uWSGI.
 app = connexion.FlaskApp(__name__, specification_dir=Configuration.SWAGGER_YAML_PATH, debug=True)
+CORS(app.app)
```

Comment:
Awesome! :tada: 

This looks like a wrong merge. Relevant changes are in Pipfile, and import of `flask_cors` and this initialization in this file.

Could you please try to rebase on top of the current master and apply your changes again?

Thanks! :+1: 

@goern could you try to run this in test environment and verify the issue was fixed by our new Thoth contributor?

Reply:
Should I fork the repo again and make changes?

Reply:
If you deleted your fork, please do so. To simplify PR, then you can create a branch for this change:

```console
git clone git@github.com:ram-nad/user-api.git # if you use git+ssh
cd user-api
git checkout -b cors
# make changes in this branch
git push --set-upstream origin cors
# ...and open a PR
```